homeassistant/components/modbus/modbus.py

```diff
@@ -135,7 +135,7 @@ async def async_modbus_setup(
         hubs = hass.data[DOMAIN]
         for name in hubs:
             if not await hubs[name].async_setup():
-                return False
+                return False  # pragma: no cover
```

Comment:
Why are we hiding/excluding coverage here?

Reply:
Because I have not been able to make a test that hits those 2 spots.

Reply:
That is fine, we should in that case just accept those as uncovered.